integration-tests/src/tests/client/process_blocks.rs

```diff
@@ -228,6 +228,23 @@ fn prepare_env_with_congestion(
     (env, tx_hashes)
 }
 
+/// Create a `TestEnv` with an account and a contract deployed to that account.
+fn prepare_env_with_contract(
```

Comment:
Would be great to make it just a method of the builder object! But, with the way real and kv runtimes are setup, that's not going to be trivial, so lgtm.